src/download/exchangeDownloader.ts

```diff
@@ -442,5 +443,5 @@ function getLatestReleaseVersion(versionGroup: {
     if (aMajor !== bMajor) return bMajor - aMajor;
     if (aMinor !== bMinor) return bMinor - aMinor;
     return bPatch - aPatch;
-  })[0].version;
+  })[0]?.version;
```

Comment:
Was seeing an error in isomorphic SDK where we tried to read `version` property from undefined